docs/css/styles.css

```diff
@@ -2699,8 +2699,20 @@ button, .button {
   padding-top: 0.5em;
   padding-bottom: 0.5em;
   min-height: 2em; }
+  .funnies-wrapper--input-container {
```

Comment:
What does the double dash signify? Personally I prefer single dashes for the whole thing (ie, `funnies-wrapper-input-container`) but I'd entertain alternatives if there's a compelling reason to use them.

Reply:
It's just a pattern (or a methodology) that i like to use because i think that it combines very well with SASS called [BEM](http://getbem.com/naming/ "BEM").

By the way, i used the wrong pattern. According to BEM, in this case, i should have used `.funnies-wrapper__input-container` instead of `.funnies-wrapper--input-container`

Reply:
Ok, cool. I don't really mind it, was just curious. Thanks for the link!